aviatrix/resource_aviatrix_edge_equinix.go

```diff
@@ -371,16 +365,18 @@ func marshalEdgeEquinixInput(d *schema.ResourceData) *goaviatrix.EdgeEquinix {
 		interface2 := &goaviatrix.EdgeEquinixInterface{
 			IfName:       interface1["name"].(string),
 			Type:         interface1["type"].(string),
-			Bandwidth:    interface1["bandwidth"].(int),
 			PublicIp:     interface1["wan_public_ip"].(string),
 			Tag:          interface1["tag"].(string),
 			Dhcp:         interface1["enable_dhcp"].(bool),
 			IpAddr:       interface1["ip_address"].(string),
 			GatewayIp:    interface1["gateway_ip"].(string),
 			DnsPrimary:   interface1["dns_server_ip"].(string),
 			DnsSecondary: interface1["secondary_dns_server_ip"].(string),
-			VrrpState:    interface1["enable_vrrp"].(bool),
-			VirtualIp:    interface1["vrrp_virtual_ip"].(string),
+		}
+
+		if interface1["type"].(string) == "LAN" && interface1["enable_vrrp"].(bool) {
+			interface2.VrrpState = interface1["enable_vrrp"].(bool)
+			interface2.VirtualIp = interface1["vrrp_virtual_ip"].(string)
```

Comment:
Could you explain what this change is?

Reply:
vrrp_state and the virtual_ip is only applicable to LAN interfaces. This check was added to the api a month back but never updated on the equinix tf.

Reply:
Here's the backend change:
https://github.com/AviatrixDev/cloudn/blob/4ec0266ee378f86b7f19ec63ad60641daefc3c9c/cloudx-local/avx/edge/interface.py#L253

Reply:
Seems worth having a code comment to explain the situation to future maintainers.

Reply:
Added the comment.